### ISSUE TITLE
Handle error when Origin install not found on Windows

### ISSUE THREAD

**Issue by GeckoEidechse** [R2NorthstarTools/FlightCore#31]
When on a Windows machine where Titanfall2 is **NOT** installed with either Steam or Origin, the check for Titanfall2 Origin install fails and causes a panic.

```
called `Result::unwrap()` on an `Err` value: Powershell(Output { inner: Output { status: ExitStatus(ExitStatus(1)), stdout: "", stderr: "Get-ItemProperty : Cannot find path 'HKEY_LOCAL_MACHINE\SOFTWARE\Respawn\Titanfall2\' because it does not exist.
At line:1 char:1
+ Get-ItemProperty -Path Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Respawn\ ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (HKEY_LOCAL_MACH...awn\Titanfall2\:String) [Get-ItemProperty], ItemNotFo 
   undException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetItemPropertyCommand
 
" }, success: false })
````
Link to crash report (private): https://sentry.io/organizations/northstar-kv/issues/3673655072/?project=6692177&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d

**Comment by Alystrasz** [comment]
This registry key is only set by Origin installs?

**Comment by GeckoEidechse** [comment]
I assumed so ^^

**Comment by GeckoEidechse** [comment]
Should be fixed with #99
